app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -125,7 +161,16 @@ export function ProjectsPageContent({ timeRange }: { timeRange: TimeRange }) {
   );
 
   return (
-    <Flex direction="column" flex="1 1 auto">
+    <div
+      css={css`
+        flex: 1 1 auto;
+        flex-direction: column;
```

Comment:
is this needed?

Reply:
addressed in 122de8e8129e41353cfc203bd03317f38e6b1ed6